src/map/controls/queryby.js

```diff
@@ -618,11 +610,10 @@ export class QueryBy extends InteractionControl {
             // Catalog layers (TOC) properties that need to be satisfied
             layersFilterObject: {
               SELECTED_OR_ALL: true, // selected or all
-              FILTERABLE:      true, // see: src/app/core/layers/layer.js#L925
+              QUERABLE:        true, // see: src/app/core/layers/layer.js#L925
```

Comment:
I'm not native speaker, but this seems like incorrect spelling.
```suggestion
              QUERYABLE:        true, // see: src/app/core/layers/layer.js#L925
```

Reply:
@alexbruy you're right, it's mispelled (we are aware of it), but for now the parameter has that name.